optimum/exporters/onnx/base.py

```diff
@@ -821,24 +821,24 @@ def __init__(
             preprocessors=preprocessors,
         )
         self._behavior = behavior
-        self.override_attributes_for_behavior()
 
-    def override_attributes_for_behavior(self):
-        """Override this to specify custom attribute change for a given behavior."""
+        # Override this to specify custom attribute change for a given behavior.
         if self._behavior is ConfigBehavior.ENCODER:
             self.task = "feature-extraction"
             self.use_past_in_inputs = False
             self.use_present_in_outputs = False
         if self._behavior is ConfigBehavior.DECODER:
             self.use_past_in_inputs = self.use_past
-            self.use_present_in_outputs = True
+            if use_present_in_outputs is None:
+                self.use_present_in_outputs = True
```

Comment:
Could this be removed as well (to keep [`use_past`](https://github.com/huggingface/optimum/blob/v1.12.0/optimum/exporters/onnx/base.py#L503) default), also not sure to understand the point of `use_present_in_outputs`, when should it be different from `use_past` ? cc @michaelbenayoun worked on https://github.com/huggingface/optimum/commit/2a4b790be4ac1640e01f3f54b1381e4b594d1695

Reply:
As discussed offline with Michael we can remove it.